buildSrc/src/main/groovy/library.java-conventions.gradle

```diff
@@ -12,6 +12,8 @@ compileJava {
     // when only a more recent JDK is available (for example, compile for Java 11 with Java 17 or 21).
     sourceCompatibility = '1.8'
     targetCompatibility = '1.8'
+    options.debug = true
```

Comment:
these need comments or documentation links

Reply:
Added comments to the .gradle file. These options are required to support the debugger "Pause" action during a WRIMS GUI model run.